docker/entrypoint.sh

```diff
@@ -19,23 +35,28 @@ else
   ipfs config Addresses.Gateway /ip4/0.0.0.0/tcp/8080
 fi
 
+echo "Confirming experimental container"
 ipfs daemon --migrate=true &
 sleep 3
 
 # Test whether the mounted directory is writable for us
-if [ ! -w "$IPFS_CLUSTER_PATH" 2>/dev/null ]; then
-  echo "error: $IPFS_CLUSTER_PATH is not writable for user $user (uid=$(id -u $user))"
+if [ ! -w "$cluster_repo" 2>/dev/null ]; then
+  echo "error: $cluster_repo is not writable for user $user (uid=$(id -u $user))"
   exit 1
 fi
 
 ipfs-cluster-service --version
 
-if [ -e "$IPFS_CLUSTER_PATH/service.json" ]; then
+if [ -e "$cluster_repo/service.json" ]; then
     echo "Found IPFS cluster configuration at $IPFS_CLUSTER_PATH"
 else
     ipfs-cluster-service init
     sed -i 's/127\.0\.0\.1\/tcp\/9094/0.0.0.0\/tcp\/9094/' "$IPFS_CLUSTER_PATH/service.json"
     sed -i 's/127\.0\.0\.1\/tcp\/9095/0.0.0.0\/tcp\/9095/' "$IPFS_CLUSTER_PATH/service.json"
 fi
 
-exec ipfs-cluster-service $@
+ipfs-cluster-service $@ &
+
+# Parent process blocks forever. Least ugly solution (still ugly)
+echo "Daemons launched"
+exec tail -f /dev/null
```

Comment:
I'm happy to use a pipe, however I am looking at the logs for a container running this image and I do see the cluster output.  Should we include a pipe anyway?  It does seem less arbitrary to tail cluster output than dev/null